integration/k8/geopm-prometheus-pkg.Dockerfile

```diff
@@ -1,27 +1,31 @@
-FROM ubuntu:22.04 AS env
+FROM ubuntu:24.04 AS env
 
 ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get update
-RUN apt-get install -yq software-properties-common python3-pip git build-essential libcap-dev \
-                        libnvidia-ml-dev libgrpc-dev libgrpc++-dev libprotobuf-dev libprotoc-dev \
-			libsystemd-dev liburing-dev libtool pkgconf protobuf-compiler \
-			protobuf-compiler-grpc unzip zlib1g-dev python3-all python3-setuptools \
-			python3-setuptools-scm wget debhelper-compat dh-python curl zstd python3-cffi
-RUN python3 -m pip install build
-RUN apt-get install -yq gpg
-RUN wget -qO - https://repositories.intel.com/gpu/intel-graphics.key | gpg --yes --dearmor --output /usr/share/keyrings/intel-graphics.gpg
-RUN echo "deb [arch=amd64,i386 signed-by=/usr/share/keyrings/intel-graphics.gpg] https://repositories.intel.com/gpu/ubuntu jammy client" | \
-  tee /etc/apt/sources.list.d/intel-gpu-jammy.list
-RUN apt-get update
-RUN apt-get install -yq libze1 libze-dev
+RUN apt-get update && apt-get install -yq --no-install-recommends \
+    software-properties-common python3-pip git build-essential libcap-dev \
+    libnvidia-ml-dev libgrpc-dev libgrpc++-dev libprotobuf-dev libprotoc-dev \
+    libsystemd-dev liburing-dev libtool pkgconf protobuf-compiler \
+    protobuf-compiler-grpc unzip zlib1g-dev python3-all python3-setuptools \
+    python3-setuptools-scm wget debhelper-compat dh-python curl zstd \
+    python3-cffi libze1 libze-dev python3-dev python3-defusedxml \
+    python3-build fakeroot
+RUN useradd -ms /bin/bash build
+USER build
+WORKDIR /home/build
 RUN git clone https://github.com/geopm/geopm.git
```

Comment:
This is buggy; it always builds the default branch no matter what version is checked out.

Reply:
Once the GRPC features are in the release, the geopm-prometheus-pkg.Docker  will go away entirely, and we will just pull the packages in from launchpad. This build docker file is a work around until we have a release with the features.  The reason it is pointing to our default branch is that this is as close to a release as we can muster at this point.

Too bad it is so hard to get a release out!

Reply:
https://geopm.github.io/install.html#install-latest-stable-release

Reply:
@bnevis-i Thanks for the long form feedback via email about how to implement the feature.  I've tried to implement it with the last patch on the stack.

Reply:
https://github.com/geopm/geopm/pull/3793/commits/67758014622358509fa86a00901d5f0e0f399095